### PR TITLE
Feature/improve springboard logs/166412220

### PR DESCRIPTION
**Pull request by truenito**
Saving parameters exported to Springboard and adding a way to visualize them.

### CODE REVIEW

File: app/jobs/spree_springboard/invoice_order_job.rb [hunk 1]
```diff
@@ -8,7 +8,8 @@ def perform(order)
         order.springboard_invoice!
       end
     rescue StandardError => error
-      ExceptionNotifier.notify_exception(error, data: { msg: "Order #{order.number}" })
+      Raven.extra_context(order_number: order.number)
+
```
Comment (mike809) [inline app/jobs/spree_springboard/invoice_order_job.rb]: Extra line here

File: app/jobs/spree_springboard/update_gift_card_job.rb [hunk 2]
```diff
@@ -5,7 +5,8 @@ class UpdateGiftCardJob < ApplicationJob
     def perform(gift_card)
       gift_card.springboard_adjust_balance!
     rescue StandardError => error
-      ExceptionNotifier.notify_exception(error, data: { msg: "Update GiftCard #{gift_card.code}" })
+      Raven.extra_context(updated_gift_card: gift_card.code)
+
```
Comment (mike809) [inline app/jobs/spree_springboard/update_gift_card_job.rb]: Empty line here

File: app/model/spree/springboard_log.rb [hunk 3]
```diff
@@ -32,11 +32,13 @@ def self.create_log(message_type, message, resource = nil, params = {})
       log_params = params.merge(
         message: message,
         message_type: message_type,
-        resource: resource
+        resource: resource,
```
Comment (mike809) [inline app/model/spree/springboard_log.rb]: Extra comma here

File: lib/spree_springboard/resource/export/order.rb [hunk 4]
```diff
@@ -34,10 +34,19 @@ def after_sync(order)
         def springboard_invoice!(order)
           if springboard_can_invoice?(order)
             invoice_springboard_id = springboard_invoice_create!(order)
-
             springboard_invoice_line_items_create!(order, invoice_springboard_id) if invoice_springboard_id.present?
             springboard_invoice_complete!(order) if invoice_springboard_id.present?
             invoice_springboard_id
+          # If the order has been invoiced AND it's open:
+          # attempt to resync line_items and complete the order.
+          elsif springboard_invoiced(order)
```
Comment (mike809) [inline lib/spree_springboard/resource/export/order.rb]: Is this duplicated from the other PR?

Comment (truenito) [reply lib/spree_springboard/resource/export/order.rb]: Yes, it has been resolved after rebasing off master and merging: https://github.com/JenniKayne/spree_springboard/pull/8/commits/cd396a13427b46a2578ee0cef50db12f50160878

**Pull request merged**